core/desugarModules.ml

```diff
@@ -212,9 +212,10 @@ and perform_renaming module_table path term_ht type_ht =
     method! bindingnode = function
       | Module (n, bs) ->
           (self, Module (n, bs))
-      | AlienBlock ab ->
-          (self, AlienBlock ab)
-      | Foreign f -> (self, Foreign f)
+      | AlienBlock (lang, lib, decls) ->
```

Comment:
An opportunity for a micro-optimisation:
```ocaml
| (AlienBlock _) as ab -> (self, ab)
```
This generates slightly better code, as `AlienBlock` is reused rather than recreated.